nix_update/version/gitlab.py

```diff
@@ -52,7 +52,10 @@ def fetch_gitlab_snapshots(url: ParseResult, branch: str) -> list[Version]:
     resp = urllib.request.urlopen(gitlab_url)
     commits = json.load(resp)
 
-    versions = fetch_gitlab_versions(url)
+    try:
+        versions = fetch_gitlab_versions(url)
+    except VersionError:
+        versions = []
     latest_version = versions[0].number if versions else "0"
```

Comment:
I think if we don't have any versions than it should be `unstable-{date}` rather than `0-unstable-{date}`.

Reply:
Isn't `0-unstable-{date}` recommended by the Nixpkgs docs? AFAIK the GitHub fetchers do the same.